cluster/manifests/audittrail-adapter/daemonset.yaml

```diff
@@ -25,8 +25,10 @@ spec:
           requiredDuringSchedulingIgnoredDuringExecution:
             nodeSelectorTerms:
             - matchExpressions:
-              - key: master
```

Comment:
Just note that this is also what we use in the skipper daemonset, I did it for consistency, maybe you can change it there as well (https://github.com/zalando-incubator/kubernetes-on-aws/blob/7acbcb75a1e30cba5c51254954fd231774a40b45/cluster/manifests/skipper/daemonset.yaml#L33 )

Reply:
Done #776